spec/commands/order/get_all_orders_command_spec.rb

```diff
@@ -32,4 +34,26 @@
     list_all_orders = "fabien: fish\nwill: burger"
     expect(response).to eq(list_all_orders)
   end
+
+  it "return only the orders of the current week" do
+    Helper.order({user_id: "asdf", user_name: "will", user_message: "burger"})
+    Helper.order({user_id: "qwer", user_name: "fabien", user_message: "fish"})
+
+    previous_week_order = Order.create(
+      :user_name => "james",
+      :lunch => "rice",
+      :date => previous_week
+    )
+    previous_week_order.save
+
+    response = get_all_orders_command.run()
+    list_all_orders = "fabien: fish\nwill: burger"
+    expect(response).to eq(list_all_orders)
+  end
+
+  private
+
+  def previous_week
```

Comment:
In case that's needed more than two times it might be a good idea to move it to the `Days` class.